app/models/instance.py

```diff
@@ -3,36 +3,38 @@
 from .. import db
 
 
+# TODO: what is this for??
 def filter(source):

```

Comment:
this filter method is used in the sanitized _name method, but i dont think we care about it any longer